lib/rails-cloud-tasks.rb

```diff
@@ -25,4 +25,14 @@ def self.configure
   def self.config
     @config ||= Configuration.new
   end
+
+  def self.queue_adapter
+    Adapter.new
+  rescue StandardError => e
```

Comment:
I think we shouldn't rescue this in any case. It's up to the developer to choose the adapter of their liking, and set it up as required.

Reply:
@guilhermearaujo I think for this case the developer does not have many options. The adapter has been already chosen, it's `RailsCloudTasks::Adapter`. Once it has defined our client must fallback if something got wrong.

I think the raised exception is about the GCP connection. Do you prefer to change to a specific one? (probably some GCP Error Class)

Reply:
In my view, this gem requires some authentication to work, and if the user fails to provide them, it should be explicit about it, and be consistent for every environment.

I, personally, would rather let the exception be thrown even on development.

Otherwise, at least raise the log level to warning and be explicit of which gem is logging it.

e.g.  `logger.warn('rails-cloud-tasks: unable to setup adapter, falling back to :inline')

And also log the original exception